usaspending_api/api_contracts/contracts/v2/search/spending_by_award.md

```diff
@@ -13,23 +13,25 @@ This endpoint takes award filters and fields, and returns the fields of the filt
     + Attributes (object)
         + `filters` (required, FilterObject)
         + `fields` (required, SpendingByAwardFields)
-        + `limit`: 60 (optional, number)
+        + `limit` (optional, number)
             How many results are returned. If no limit is specified, the limit is set to 10.
-        + `order`: `desc` (optional, string)
+        + `order` (optional, string)
```

Comment:
Could use `enum[string]` with `Members` `asc` and `desc` and the API Blueprint `Default` format to more formally convey what's specified in this description.